lib/services.rb

```diff
@@ -17,7 +17,7 @@ def self.imminence_api
   end
 
   def self.worldwide_api
-    @worldwide_api ||= GdsApi::Worldwide.new(Plek.new.find("whitehall-admin"))
+    @worldwide_api ||= GdsApi::Worldwide.new(Plek.new.find("whitehall-frontend"))
```

Comment:
```suggestion
    @worldwide_api ||= GdsApi.worldwide
```

Since https://github.com/alphagov/gds-api-adapters/pull/980 we can rely on that being configured to talk to whitehall-frontend and it makes it easier to change it again in the future across all our apps.